stacker/providers/aws.py

```diff
@@ -183,3 +183,27 @@ def get_outputs(self, stack_name, *args, **kwargs):
             stack = self.get_stack(stack_name)
             self._outputs[stack_name] = get_output_dict(stack)
         return self._outputs[stack_name]
+
+    def get_stack_info(self, stack_name):
+        """ Get the template and parameters of the stack currently in AWS
+
+        Returns [ template, parameters ]
+        """
+        try:
+            stacks = retry_on_throttling(
+                self.cloudformation.describe_stacks,
+                kwargs=dict(stack_name_or_id=stack_name))
+            stack = stacks[0]
+            parameters = dict()
+            for p in stack.parameters:
+                parameters[p.key] = p.value
+            ret = stack.get_template()
+            template = ret['GetTemplateResponse']['GetTemplateResult']
+            template = template['TemplateBody']
+            return [template, parameters]
+        except boto.exception.BotoServerError as e:
```

Comment:
This also seems pretty broad - you really only need this around the self.cloudformation.describe_stacks call, right?